quail/processes/wps_climdexInput_raw.py

```diff
@@ -154,10 +162,20 @@ def __init__(self):
     def generate_dates(
         self, request, filename, obj_name, date_fields, date_format, cal
     ):
-        load_rdata_to_python(filename, obj_name)
-        return robjects.r(
-            f"as.PCICt(do.call(paste, {obj_name}[,{date_fields}]), format='{date_format}', cal='{cal}')"
-        )
+        load_rda(filename, obj_name)
+        try:
+            return robjects.r(
+                f"as.PCICt(do.call(paste, {obj_name}[,{date_fields}]), format='{date_format}', cal='{cal}')"
+            )
+        except RRuntimeError as e:
+            raise ProcessError(msg="Error generating dates")
+
+    def column(self, df_name, column_name, var):
+        df_column = robjects.r(df_name).rx2(column_name)
+        if robjects.r["is.null"](df_column)[0]:
+            raise ProcessError(f"RRuntimeError: No {var} column of that name")
```

Comment:
Again, we probably don't want to list `RRuntimeError` if we aren't actually catching that error.